gradlew.bat

```diff
@@ -30,7 +30,7 @@ set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%
 
 @rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS=-Dfile.encoding=UTF-8
+set DEFAULT_JVM_OPTS=-Dfile.encoding=UTF-8 "-Xmx64m" "-Xms64m"
```

Comment:
is this too low?

Reply:
Yeah but thats the [current default in there generated files][1]

[1]: https://github.com/gradle/gradle/blob/362226499afd9180ebc19822d44fca5fbaf004e9/gradlew#L47

Reply:
If a different heap size is appropriate, we can change it.  It's fine for us to diverge from the upstream gradlew[.bat] as needed.

Reply:
For now build seems to work fine. If we see any issue then we can change. I believe Gradle wrapper internally forks a new jvm process so it needs bare minimum memory for its work